src/main/scala/ru/otus/sc/game/dao/GameCredentialsDao.scala

```diff
@@ -0,0 +1,54 @@
+package ru.otus.sc.game.dao
+
+import ru.otus.sc.game.model.{Player, User}
+
+trait GameCredentialsDao {
+
+  /**
+    * Получить список всех пользователей
+    *
+    * @return список всех пользователей
+    */
+  def allUsers(): List[User]
```

Comment:
```suggestion
  def allUsers: List[User]
```

Скобочки часто используются у методов где нам надо обозначить что у них будет обязательно какой-то сайд-эффект, типа `destroy()`. В этом случае это просто что-то вроде геттера (хоть и с потенциальным сайд-эффектом)

Reply:
Вот это, на самом деле, заставила сделать Idea. Изначально они были убраны.
Я так и не понял как она вывела у себя, что метод имеет сайд-эффект.